scripts/items/armor/rf_breastplate.nut

```diff
@@ -0,0 +1,30 @@
+this.rf_breastplate <- this.inherit("scripts/items/armor/armor", {
+	m = {},
+	function create()
+	{
+		this.armor.create();
+		this.m.ID = "armor.body.rf_breastplate";
+		this.m.Name = "Breastplate";
+		this.m.Description = "A quality breastplate and fauld over a simple gambeson offering strong protection.";
+		this.m.SlotType = this.Const.ItemSlot.Body;
```

Comment:
replace `this.Const` with `::Const`.
This is the first occurence that I saw and I will not mark all the other ones